code/modules/reagents/chemistry/reagents/toxins.dm

```diff
@@ -502,6 +502,19 @@
 	M.adjustToxLoss(1)
 	..()
 
+/datum/reagent/somnistimine
+	name = "Somnistimine"
+	id = "somnistimine"
+	description = "A powerful tranquilizer."
+	reagent_state = LIQUID
+	color = "#2E2E61"
+	metabolization_rate = 0.4
+
+/datum/reagent/somnistimine/on_mob_life(mob/living/M)
+	if(current_cycle>= 3)
```

Comment:
I understand yer point of view Fox, but once you start mixing chems, everything chances. Once you mix chems, you are sacrificing effectivness of already present chems by tweaking amounts and effectivness of the delivery method. Taking down one person is easy, but once ye start thinking of the bigger picture, storage space is finite and reloading/refilling syringe gun/parapen becomes a hassle. 

A long time ago, there was a PR/idea to separate science job into "sub-jobs". RnD/Robotics and SciChem/Toxins. Even a map was made for both. Dividing science in half would cut down the "Science antag bias" crap and make science weaker and not such a obvious choice for antags. This would also limit the access to syringe guns and other crap. Going off topic, but the "memechems" comes strait from the scichem department side since they have access and the possibilty to get most out of chems. Maybe instead of nerfing chems to the ground, nerf the chems a bit and start limiting the access to them and delivery methods to further make it harder to obtain them.